x/projects/client/cli/tx_set_admin_policy.go

```diff
@@ -13,13 +13,13 @@ import (
 
 var _ = strconv.Itoa(0)
 
-func CmdSetAdminPolicy() *cobra.Command {
+func CmdSetPolicy() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "set-admin-policy [project-index] [policy-file-path]",
+		Use:   "set-policy [project-index] [policy-file-path]",
 		Short: "set policy to a project",
-		Long:  `The set-admin-policy command allows a project admin to set a new policy to its project. The policy file is a YAML file (see cookbook/project-policies/example.yml for reference). The new policy will be applied from the next epoch.`,
+		Long:  `The set-policy command allows a project admin to set a new policy to its project. The policy file is a YAML file (see cookbook/project-policies/example.yml for reference). The new policy will be applied from the next epoch.`,
```

Comment:
The path to the example policy file is old. The right path is cookbook/projects/example_policy.yml